tests/src/test/java/alluxio/shell/command/UnsetTtlCommandIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Test for unsetTtl command.
  */
-public final class UnsetTtlTest extends AbstractAlluxioShellTest {
+public final class UnsetTtlCommandIntegrationTest extends AbstractAlluxioShellTest {
```

Comment:
The root of the problem is that you're making a pull request from your `master` branch. Whenever you make a pull request in Github, you should create a new branch just for that pull request. To create a new `rmr` branch, you can do

```bash
# Nickname the Alluxio/alluxio repository to "upstream"
git remote add upstream git@github.com:Alluxio/alluxio.git
# Get the latest updates from Alluxio/alluxio
git fetch upstream
git checkout upstream/master
# Create a local branch named rmr
git checkout -b rmr
```

Once you've made your changes, you can push the branch to your Github fork (dayhost/alluxio) by running

```bash
git push origin rmr
```

Once you've pushed the branch to your fork, you can use the Github UI to create pull request for that branch.

Before doing all of this, make sure your email is set up correctly by running

```
git config --global user.email "dayhost@users.noreply.github.com"
```